app/src/main/java/com/youarelaunched/challenge/ui/screen/view/VendorsScreen.kt

```diff
@@ -23,36 +24,64 @@ fun VendorsRoute(
 ) {
     val uiState by viewModel.uiState.collectAsState()
 
-    VendorsScreen(uiState = uiState)
+    VendorsScreen(
+        uiState = uiState,
+        onSearchQueryChange = { viewModel.onSearchQueryChange(it) },
+        onSearchClick = { viewModel.onSearchClick() }
```

Comment:
Я бы так писал, короче, и даже на оптимизацию компоуза влияет по-моему
```suggestion
        onSearchClick = viewModel::onSearchClick
```